test/exclude-system-headers.cpp

```diff
@@ -1,4 +1,4 @@
-// RUN: %clang -fsyntax-only -std=c++11 -w -ferror-limit=0 -Xclang -verify -Xclang -load -Xclang %llvmshlibdir/misracpp2008.so -Xclang -plugin -Xclang misra.cpp.2008 -Xclang -plugin-arg-misra.cpp.2008 -Xclang all %s
+// RUN: %clang -isystem -fsyntax-only -std=c++11 -w -ferror-limit=0 -Xclang -verify -Xclang -load -Xclang %llvmshlibdir/misracpp2008%pluginext -Xclang -plugin -Xclang misra.cpp.2008 -Xclang -plugin-arg-misra.cpp.2008 -Xclang all %s
```

Comment:
This kills the tests on my local machine. Why exactly do you need the "isystem" flag?

To me it looks like an error that swallows the -fsyntax-only option.